src/components/DatePickerDocked/HeaderItem.tsx

```diff
@@ -0,0 +1,119 @@
+import { useComponentStyles } from '@bambooapp/bamboo-atoms';
+import { memo, useCallback, useMemo } from 'react';
+import { useMolecules } from '../../hooks';
+import { useStore } from './DatePickerDocked';
+
+function HeaderItem({
+    value,
+    onNext,
+    onPrev,
+    selecting,
+    type,
+}: {
+    value: number | string;
+    onNext: () => any;
+    onPrev: () => any;
+    type: 'month' | 'year' | undefined;
+    selecting: boolean;
+}) {
+    const { View, IconButton, TouchableRipple, Text } = useMolecules();
+    const [{ startDateYear, endDateYear, pickerType }, setStore] = useStore(state => state);
+    const headerItemStyles = useComponentStyles('DatePicker_HeaderItem');
+    const disabled = pickerType && pickerType !== type;
+
+    const {
+        buttonContainerStyle,
+        buttonWrapperStyle,
+        spacerStyle,
+        buttonStyle,
+        innerStyle,
+        labelStyle,
+        iconContainerStyle,
+        emptyViewStyle,
+    } = useMemo(() => {
+        const {
+            buttonContainer,
+            buttonWrapper,
+            spacer,
+            buttonStyle: _buttonStyle,
+            innerStyle: _innerStyle,
+            labelStyle: _labelStyle,
+            emtpyView,
+        } = headerItemStyles;
+
+        return {
+            buttonContainerStyle: buttonContainer,
+            buttonWrapperStyle: buttonWrapper,
+            spacerStyle: spacer,
+            buttonStyle: _buttonStyle,
+            innerStyle: _innerStyle,
+            labelStyle: _labelStyle,
+            iconContainerStyle: { opacity: 1 },
+            emptyViewStyle: emtpyView,
+        };
+    }, [headerItemStyles]);
+
+    const handlePressDropDown = useCallback(() => {
+        setStore(prev => ({
+            ...prev,
+            pickerType: pickerType ? undefined : type,
+        }));
+    }, [setStore, type, pickerType]);
+
+    return (
+        <View style={buttonContainerStyle} pointerEvents={'box-none'}>
+            <View style={spacerStyle} pointerEvents={'box-none'} />
+            {!selecting ? (
+                <View style={buttonWrapperStyle}>
+                    <IconButton
+                        type="material-community"
+                        name="chevron-left"
+                        size="sm"
+                        // Todo: Translate
+                        accessibilityLabel={'Previous'}
+                        onPress={onPrev}
+                        disabled={value === startDateYear}
+                    />
+                </View>
+            ) : (
+                <View style={emptyViewStyle} />
```

Comment:
I suppose this can be dropped now, right?

Reply:
Updated code is in the new PR